utils/cdk-docker-execute.sh

```diff
@@ -151,8 +153,8 @@ if [ "${#cfn_files[@]}" -gt 0 ]; then
     #
     # Copy and then remove these files to avoid permission setting errors when running in a single container
     #
-    cp ${CDK_WORK_DIR}/cdk.out/CdkNagScanStack.template.json ${_ASH_OUTPUT_DIR}/${DIRECTORY}/${cfn_filename}_cdk_nag_results/
```

Comment:
I'm getting errors when running ASH against the dummy files
cp: cannot stat '/tmp/cdk-nag-scan.8SM0u/cdk.out/*.template.json': No such file or directory
rm: cannot remove '/tmp/cdk-nag-scan.8SM0u/cdk.out/*.template.json': No such file or directory
cp: cannot stat '/tmp/cdk-nag-scan.8SM0u/cdk.out/*.template.json': No such file or directory
rm: cannot remove '/tmp/cdk-nag-scan.8SM0u/cdk.out/*.template.json': No such file or directory
cp: cannot stat '/tmp/cdk-nag-scan.8SM0u/cdk.out/*.template.json': No such file or directory
rm: cannot remove '/tmp/cdk-nag-scan.8SM0u/cdk.out/*.template.json': No such file or directory

Reply:
@begimher Just seeing this, apologies on the delayed response! Can you try running with `--debug` included on that same directory? It seems like it's taking the wildcard literally in the path instead of expanding it. The duplication of the log points at expansion still happening. I'm not able to replicate the issue on my end with the same scan dir so far

Reply:
@begimher found the issue impacting this (expansion of wildcard paths happening within the loop), fix has been pushed up to the source branch.